include/arch/arm/cortex_m/mpu/arm_mpu.h

```diff
@@ -80,7 +80,14 @@ struct arm_mpu_config {
 		.attr = _attr, \
 	}
 
-/* Reference to the MPU configuration */
+/* Reference to the MPU configuration.
+ *
+ * It shall be defined and populated for each SOC (in SOC definition). It
```

Comment:
Rather stilted English.  Can this be written as:

    This struct is defined and populated for each SoC (in the SoC definition),
    and holds the build-time configuration information for the fixed MPU
    regions enabled during kernel initialization.  Dynamic regions modified
    during runtime are not kept here.

If that's OK, can you add where these dynamic regions are kept?

Reply:
Or can zephyr support dynamic MPU regions un arm soc now?

Reply:
@dbkinder , this is OK. Dynamic regions, however are configured on the fly and are not stored anywhere, at the moment.